### PR TITLE
Add tab completion to generate and destroy

### PR DESCRIPTION
**Pull request by a3ammar**
Hi,
Always bugged me that using the command line was a better experience than using `projectile-rails-generate` so I made it better. generate now completes common generators and destroy when you select a generator it give you choices of available resources (this doesn't change the old behavior).

Destroy tests are lacking because I couldn't figure out how test the choices of resource you get once you choose a generator.

I'm not good at elisp so if this can be improved please let me know how.

### CODE REVIEW

File: projectile-rails.el [hunk 1]
```diff
@@ -204,6 +204,20 @@
 (defvar-local projectile-rails-zeus-sock nil
   "The path to the Zeus socket file")
 
+(defvar projectile-rails-generators
+  '(("assets" (("app/assets/javascripts/" "app/assets/javascripts/\\(.+\\)\\.coffee$")))
```
Comment (asok) [inline projectile-rails.el]: @a3ammar I guess `"assets"` is one of the possible completion. But what's the idea behind `(("app/assets/javascripts/" "app/assets/javascripts/\\(.+\\)\\.coffee$"))`? And why only `coffee`?
Sorry I did try to find information on that but I failed.

**Comment by asok** [comment]
It's a really neat addition. Thanks. I've left some comment in the code.

File: projectile-rails.el (hunk 1 above)
Comment (a3ammar) [inline projectile-rails.el]: You are right, it should be changed to: 
`(("app/assets/" "app/assets/\\(?:stylesheets\\|javascripts\\)/\\(.+?\\)\\..+$))`
I will modify that.

**Comment by a3ammar** [comment]
I'm not totally sure if you get notified when I push commits to this pull request. If not, I've corrected the `assets` regexp.

**Comment by asok** [comment]
Oh I haven't receive any notification. Merged it now thanks.

**Pull request merged**